sample_configuration.properties

```
@@ -2,17 +2,17 @@ api.url={API_URL}
 database.url={DATABASE_URL}
 database.username={DATABASE_USERNAME}
 database.password={DATABASE_PASSWORD}
-institution.super_admin_user_one.email_address={INSTITUTION_SUPER_ADMIN_USER_ONE_EMAIL_ADDRESS}
-institution.super_admin_user_one.password={INSTITUTION_SUPER_ADMIN_USER_ONE_PASSWORD}
+afet_yonetim_sistemi_admin.email_address={AFET_YONETIM_SISTEMI_ADMIN_EMAIL_ADDRESS}
+afet_yonetim_sistemi_admin.password={AFET_YONETIM_SISTEMI_ADMIN_PASSWORD}
```

@cagla88 
Aşağıdaki isimlendirme standartlarında ilerlememiz daha iyi olabilir;

- `{institution_name}.{field}`
- `{institution_name}.{user_definition}.{field}`

```suggestion
afet_yonetim_sistemi.id={AFET_YONETIM_SISTEMI_ID}
afet_yonetim_sistemi.admin.email_address={AFET_YONETIM_SISTEMI_ADMIN_EMAIL_ADDRESS}
afet_yonetim_sistemi.admin.password={AFET_YONETIM_SISTEMI_ADMIN_PASSWORD}
```